### PR TITLE
docs: Presentation Exchange - adds samples

### PR DESCRIPTION
**Pull request by soluchok**
Adds Presentation Exchange samples.

Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

### CODE REVIEW

File: pkg/doc/presexch/README.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      "id": "2dc74354-e965-4883-be5e-bfec48bf60c7",
+      "issuer": "",
+      "type": "VerifiableCredential"
+    }
+  ]
+}
+```
+
+As you can see the VP has a credential without `first_name` and `last_name` (because of `limit_disclosure`).
+Also, instead of `age`, we have a boolean value (because of `predicate`).
```
Comment (troyronda) [inline pkg/doc/presexch/README.md:103]: are you providing any examples of using the presentation exchange code in aries-framework-go?

Comment (soluchok) [reply pkg/doc/presexch/README.md:103]: Done!

Comment (troyronda) [reply pkg/doc/presexch/README.md:103]: Can we add a link from this document to the code examples?

**Review by troyronda — APPROVED** [review]

File: pkg/doc/presexch/README.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  "type": [
+    "VerifiablePresentation",
+    "PresentationSubmission"
+  ],
+  "verifiableCredential": [
+    {
+      "@context": [
+        "https://www.w3.org/2018/credentials/v1"
+      ],
+      "age": true,
```
Comment (llorllale) [inline pkg/doc/presexch/README.md:88]: @soluchok 

This is a different claim than the one originally issued (by the Issuer). This new claim is not covered by cryptographic proof from the Issuer; it is unverifiable.

How will this work in practice?

File: pkg/doc/presexch/README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+# Presentation Exchange
+
+This document shows how to use [Presentation Exchange](https://identity.foundation/presentation-exchange) by examples.
+Code examples can be found [here](example_test.go).
+
+1. This example demonstrates `predicate` and `limit_disclosure` usage.
+   The `presentation definition` below requires field `age` to be greater or equal to 18.
+   Also, we have `limit_disclosure=true` which requires that output data is limited to the entries specified in the `fields` property.
+   The `predicate` means that the result should be expressed as a boolean value.
```
Comment (llorllale) [inline pkg/doc/presexch/README.md:9]: @soluchok 

> The `predicate` means that the result should be expressed as a boolean value.

This is a misunderstanding of the spec. The predicate is just criteria that the credential submitted in the presentation must satisfy. It is _not_ a transformation function.

**Pull request merged**